config/migrate.go

```diff
@@ -24,6 +24,10 @@ func Migrate(from, to string) error {
 	files := []string{*sourceConfig.SamlSettings.IdpCertificateFile, *sourceConfig.SamlSettings.PublicCertificateFile,
 		*sourceConfig.SamlSettings.PrivateKeyFile}
 
+	for _, pk := range sourceConfig.PluginSettings.SignaturePublicKeyFiles {
```

Comment:
@iomodo do we need this if we're invoking https://github.com/mattermost/mattermost-server/pull/12390/files#diff-d07092579ced8908e9521d5930e3328aR132

Reply:
Correct, we do not need that.
I updated the public key as well

Reply:
Looks like this change is still outstanding.